ecmascript/parser/src/parser/typescript.rs

```diff
@@ -456,19 +459,6 @@ impl<'a, I: Tokens> Parser<'a, I> {
         })
     }
 
-    fn parse_ts_type_predicate_prefix(&mut self) -> PResult<'a, Option<Ident>> {
```

Comment:
I ended up removing this because I needed to check for the `is` keyword above and ended up changing the code to clearly return when it's not a type predicate.